ManageIQ/integration_tests
# [WIP]Modify some functions in gen_data to accept kwargs, redistribute reports test cases and automate a test

This PR brings the following changes:
1.  Add 2 new test cases
    1. `test_non_admin_user_reports_access_rest`
    2. `test_reports_create_schedule_send_report`
2. Redistribute report schedules test cases and make the following changes.
    1. Remove yaml files inside `schedule_reports` and modify the fixtures `schedule_data` of `test_report_schedules.py` to use parameters.
    2. Put all crud tests inside `test_crud.py`.
    3. Add a new `test_reports.py` file that includes all tests related to reports which are not crud
3. Modify gen_data
    1. Remove unnecessary functions such as `arbitration_settings`, `arbitration_rules` and `blueprints` which are no longer a supported functionality.
    2. Modify `users`, `groups`, `tenants`, `roles` to accept kwargs.

{{ pytest: cfme/tests/intelligence/reports/test_reports.py cfme/tests/intelligence/reports/test_crud.py cfme/tests/intelligence/reports/test_reports_schedules.py cfme/tests/intelligence/reports/test_views.py cfme/tests/configure/test_rest_access_control.py cfme/tests/intelligence/reports/test_generate_report.py --use-template-cache -sqvvvv }}
